.github/workflows/tests.yml

```diff
@@ -0,0 +1,30 @@
+# Authored by Alexandra N. Walker <alex.walker@indicio.tech> ☀️
+
+name: Tests
+on:
+  push:
+    branches: 
+      - main
+      - feature/revocation
+      - test/tests
```

Comment:
Should probably remove these branches from the on.push condition so the tests only run on main